pom.xml

```diff
@@ -179,7 +179,13 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.10.1</version>
+            <version>2.10.1</version>
```

Comment:
Make sure this is version 2.10.2 so that it matches jackson-core.

Reply:
I want to keep jackson-databind versin 2.10.1 so I changed jackson-core version to 2.10.1.